resources/grch37/gnomad.py

```diff
@@ -0,0 +1,345 @@
+
+    if data_type == "exomes":
+        current_release = CURRENT_EXOME_RELEASE
+        releases = EXOME_RELEASES
+        releases.remove("2.1")
+    else:
+        current_release = CURRENT_GENOME_RELEASE
+        releases = GENOME_RELEASES
+
+    return VersionedTableResource(
+        current_release,
+        {release: TableResource(path=_public_coverage_ht_path(data_type, release)) for release in releases},
+    )
+
+
+def public_pca_loadings(subpop: str = "") -> TableResource:
+    """
+    Returns the TableResource containing sites and loadings from population PCA
+
+    :param str subpop: Can be empty ("") -> global, "eas" or "nfe"
+    :return: gnomAD public PCA loadings TableResource
+    :rtype: TableResource
+    """
+    if subpop not in ["", "eas", "nfe"]:
+        raise DataException(
+            'Available subpops are "eas" or "nfe", default value "" for global'
+        )
+
+    return TableResource(path=_public_pca_ht_path(subpop))
+
+
+def release_vcf_path(data_type: str, version: str, contig: str) -> str:
+    """
+    Publically released VCF. Provide specific contig, i.e. "20", to retrieve contig
+    specific VCF
+    :param str data_type: One of "exomes" or "genomes"
+    :param str version: One of the release versions of gnomAD on GRCh37
+    :param str contig: Single contig "1" to "Y"
+    :return: Path to VCF
+    :rtype: str
+    """
+    contig = f".{contig}" if contig else ""
+    return f"gs://gnomad-public/release/{version}/vcf/{data_type}/gnomad.{data_type}.r{version}.sites{contig}.vcf.bgz"
+
+
+def metadata(data_type: str) -> VersionedTableResource:
+    """
+    Sample metadata associated with gnomAD release
+
+    :param str data_type: One of "exomes" or "genomes"
+    :return: Versioned metadata Tables
+    :rtype: VersionedTableResource
+    """
+    if data_type not in DATA_TYPES:
+        raise DataException(f'{data_type} not in {DATA_TYPES}')
+
+    if data_type == "exomes":
+        current_meta = CURRENT_EXOME_META
+        dates = EXOME_META_DATES
+    else:
+        current_meta = CURRENT_GENOME_META
+        dates = GENOME_META_DATES
+
+    return VersionedTableResource(
+        current_meta,
+        {date: TableResource(path=_metadata_ht_path(data_type, date)) for date in dates},
+    )
+
+
+def fam(data_type: str, true_trios: bool = False) -> PedigreeResource:
+    """
+    Returns the path to gnomAD pedigree file.
+
+    :param str data_type: One of 'exomes' or 'genomes'
+    :param bool true_trios: If set, removes all families with more than one offspring
+    :return: Path to fam file
+    :rtype: str
+    """
+    if data_type not in DATA_TYPES:
+        raise DataException(f'{data_type} not in {DATA_TYPES}')
+
+    if data_type == "exomes":
+        current_fam = CURRENT_EXOME_FAM
+    else:
+        current_fam = CURRENT_GENOME_FAM
+
+    return PedigreeResource(path=_fam_path(data_type, current_fam, true_trios))
+
+
+def vep_config_path():
```

Comment:
I agree, it felt out of place putting it here but I wasn't sure where else it could live or if it was even necessary. It looks to only be used in https://github.com/macarthur-lab/gnomad_hail/blob/03b9d7e6ff7e57828f880bc74145f540c8c485b5/utils/generic.py#L271 so perhaps it should be moved into that module?

Reply:
sounds good to me!